### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AlexeyBrambalov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
 
+#### Description of the project
+
+New website that hosts Compass Collective students artworks to make these publicly available. Also build/leverage tools that enable Compass Collective to upload, amend and take down any artwork from the site. 
+
+Target Audience - publicly available.  
+
+#### Team members
+
+- Project manager -  Sonjide Hussain
```
Comment (dahfool) [inline README.md:11]: Please link names their github accounts

Comment (AlexeyBrambalov) [reply README.md:11]: done

Comment (dahfool) [reply README.md:11]: @AlexandruBudaca Please link the TAs, Sonijde and Jack as well if he has a github account.

Comment (AlexeyBrambalov) [reply README.md]: Done. I couldn't find Zach's Github account

**Pull request merged**